index.html

```diff
@@ -0,0 +1,130 @@
+                </p>
+                <h2 class="time">2023.10.15(THU) ~ 16(FRI)</h2>
+                <p>@ National Museum of Nigeria, Art Center Lagos and more</p>
+            </article>
+        </section>
+    </header>
+    <section id="main-program">
+        <h2 class="program-title wht">Main Program</h2>
+        <section id="cards">
+            <article class="card">
+                <img src="./assets/icons/home_iot_device_FILL0_wght400_GRAD0_opsz48.svg" alt="home_iot_device">
+                <h2>Home IoT</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos accusantium ipsam minima.
+                </p>
+            </article>
+            <article class="card">
+                <img src="./assets/icons/smart_outlet_FILL0_wght400_GRAD0_opsz48.svg" alt="smart_outlet">
+                <h2>Smart Outlet</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos accusantium ipsam minima.
+                </p>
+            </article>
+            <article class="card">
+                <img src="./assets/icons/assistant_on_hub_FILL0_wght400_GRAD0_opsz48.svg" alt="home_assistance">
+                <h2>Home Assistance</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos accusantium ipsam minima.
+                </p>
+            </article>
+            <article class="card">
+                <img src="./assets/icons/rocket_launch_FILL0_wght400_GRAD0_opsz48.svg" alt="rocket_science">
+                <h2>Rocket Science</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos accusantium ipsam minima.
+                </p>
+            </article>
+        </section>
+        <h2 class="Program-foot dk">SEE THE WHOLE PROGRAM</h2>
+        <h2 class="Program-foot mb">Join CC Global Summit 2023</h2>
+    </section>
+    <section id="speakers">
+
+        <h2 class="program-title">Featured Speakers</h2>
+        <section id="speaker"></section>
+        <button id="toggle-more" class="toggle-more-less mb">MORE</button>
+
+    </section>
+    <footer>
+        <section id="sponsors">
+            <h2 class="program-title wht">Partner</h2>
+            <ul class="sponsors">
+                <li>mosilla</li>
+                <li><span>Goog<span class="bd-1">l</span>e</span></li>
+                <li>NAVER</li>
+                <li>daumkakao</li>
+                <li>airbnb</li>
+            </ul>
+        </section>
+        <section class="buttom-f-nav">
+            <a class="logo logo-abt" href="#">BLK-NG</a>
+            <article class="short-f-note">
+                <h2>2023 Creative Commons Korea Some Rights Reserved.</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Modi.
+                </p>
+            </article>
+        </section>
+    </footer>
```

Comment:
- [ ] 👏👏 Excellent work on the footer! However, as per the design, the footer should be hidden in the mobile version. Currently, it is visible in all screen sizes. To match the design requirements, please update your code to hide the footer in the mobile version.

By making this adjustment, your project will have a more consistent and visually appealing layout across different screen sizes. Keep up the great work, and happy coding! 🌟💻🚀